client/src/components/payments/checkoutForm.js

```diff
@@ -0,0 +1,69 @@
+import React, { Component } from "react";
+import { pay } from '../../api/payments';
+
+import {
+  CardElement,
+  CardNumberElement,
+  CardExpiryElement,
+  CardCVCElement,
+  PaymentRequestButtonElement,
+  StripeProvider,
+  Elements,
+  injectStripe
+} from 'react-stripe-elements';
+
+import './checkoutForm.css'
```

Comment:
Just wondering why are you importing CSS instead of the css-in-js approach as we have done before, kind of a disconnect in styling. I know that is done on the profile page, but my stance on that is the same, if we are going to do one or the other, we should have it the same across the board, no?

Reply:
See above. Styling will be in issue 59. Also, I thought that whoever was doing the profile pg was going to remove the css import. I can refactor if that's not in the works.

Reply:
hmm maybe it is and I am out of touch 👍

Reply:
As for upload-icon -- I didn't do it, but can undo.

Reply:
ah ok, maybe in another ticket if you don't cause it should be in public/images